Src/Graphics/Font.cpp

```diff
@@ -72,7 +72,7 @@ void Font::renderAtlas(long chr) {
 
                 if (glyphWidth>0 && glyphHeight>0) {
                     if (buffer==nullptr) {
-                        buffer = new uint8_t[atlasDims*atlasDims*4];
+                        buffer = new uint8_t[static_cast<uint8_t>(atlasDims)^2*4];
```

Comment:
^ is the binary XOR in C/C++, there's no pow operator

Reply:
hah thanks, I ended up reverting these changes anyway so this can be closed now.